webapp/lib/model.g.dart

```diff
@@ -209,7 +209,7 @@ class MessageDirection {
   static MessageDirection fromString(String text, [MessageDirection defaultValue = MessageDirection.Out]) {
     if (text != null) {
       const prefix = 'MessageDirection.';
-      String valueName = text.startsWith(prefix) ? text.substring(prefix.length) : text;
+      var valueName = text.startsWith(prefix) ? text.substring(prefix.length) : text;
```

Comment:
has `valueName` changed type / can it be non-String now? I'm fine either way, just wondering

Reply:
@marianamarasoiu No functional or type change... just a Dart syntax style change. The prevalent Dart style is to use `var` when declaring local variables of the form `var x = something` because Dart can intuit the local variable's type from the right hand side of the assignment. That local variable has that type from that point forward and cannot be assigned something of a different type.

That said...
1) Sometimes I use the variable type on the left hand side to make it more obvious
2) If it is more clear to use the variable type here or elsewhere please let me know and I can change it.

Reply:
yup, that makes sense, thanks for the explanation! I'll also keep the recommended style in mind for future code, I haven't developed that good a relationship with `var` yet :)